packages/gcf-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcf-utils",
-  "version": "5.7.1",
+  "version": "5.5.1",
```

Comment:
That doesn't seem like a number that is supposed to go backwards

Reply:
fixed upstream.